### DISCUSSION TITLE
Account blocked, never even have i signed up

### DISCUSSION THREAD

**Discussion by rayyanekaputra** [vercel/community#3799] [Account Issue] [answered]
### Question

![image](https://github.com/vercel/community/assets/79846835/0c0766d6-b423-463c-b0b8-39daac96ef85)

did i do something wrong here?

**Comment by FilemoniXYT** [comment]
i have that too

**Comment by amyegan** [comment, marked as answer]
Hey @rayyanekaputra and @FilemoniXYT. Sorry you're having trouble signing up for a new account. Due to the sensitive nature of account information, we cannot safely discuss it in this public forum. Please email [registration@vercel.com](mailto:registration@vercel.com) and include the details of the project you want to deploy 🙏
